charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/alertmanager-overview.yaml

```diff
@@ -21,4 +21,29 @@ metadata:
-{{- end }}
\ No newline at end of file
+{{- end }}
+---
+{{- if and .Values.grafana.operator.dashboardsConfigMapRefEnabled (or .Values.grafana.enabled .Values.grafana.forceDeployDashboards) (semverCompare ">=1.14.0-0" $kubeTargetVersion) (semverCompare "<9.9.9-9" $kubeTargetVersion) .Values.grafana.defaultDashboardsEnabled (or .Values.alertmanager.enabled .Values.alertmanager.forceDeployDashboards) }}
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" $) "alertmanager-overview" | trunc 63 | trimSuffix "-" }}
+  namespace: {{ template "kube-prometheus-stack-grafana.namespace" . }}
+  labels:
+    {{- if $.Values.grafana.sidecar.dashboards.label }}
+    {{ $.Values.grafana.sidecar.dashboards.label }}: {{ ternary $.Values.grafana.sidecar.dashboards.labelValue "1" (not (empty $.Values.grafana.sidecar.dashboards.labelValue)) | quote }}
+    {{- end }}
+    app: {{ template "kube-prometheus-stack.name" $ }}-grafana
+spec:
```

Comment:
Do we need 

```
  datasources:
    - inputName: "DS_PROMETHEUS"
      datasourceName: "prom2"
```

here? As optional spec, like folder?

Ref: https://github.com/grafana/grafana-operator/blob/88f7b732f88c8453ac930efa9947ce8a42b87508/examples/datasource_mapping/resources.yaml#L59-L61

Reply:
will double check on my instance, 
but we may need to add a template related to datasource, that will be used as `Datasource`

```
  - apiVersion: grafana.integreatly.org/v1beta1
    kind: GrafanaDatasource
    metadata:
      name: prometheus
      annotations:
        argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
    spec:
      allowCrossNamespaceImport: true
      instanceSelector:
        matchLabels:
          grafana: home
      datasource:
        name: prometheus
        type: prometheus
        access: proxy
        url: http://prometheus-operated.prometheus-stack.svc.cluster.local:9090
        isDefault: true
        jsonData:
          "tlsSkipVerify": true
          "timeInterval": "5s"
        editable: true
```